Lib/pathlib/__init__.py

```diff
@@ -70,7 +70,7 @@ def __new__(cls, *args, **kwargs):
             cls = PureWindowsPath if os.name == 'nt' else PurePosixPath
         return object.__new__(cls)
 
-    def __init__(self, *args):
+    def _load_args(self, args):
```

Comment:
Add a doc string "Return ..." saying what this does and add that called in PurePathBase.__init__.